screens/Camera.js

```diff
@@ -1,149 +1,174 @@
   constructor(props) {
     super(props);
     this.state = {
       hasCameraPermission: null,
       type: Camera.Constants.Type.back,
       dataURI: [],
-      uploadStatus: ''
+      uploadStatus: '',
+      customerId: '',
+      transactionId: ''
     };
   }
-
+  
   async componentDidMount() {
     const { status } = await Permissions.askAsync(Permissions.CAMERA);
-    this.setState({ hasCameraPermission: status === 'granted' });
+    let userId = this.props.navigation.getParam('customerId', 'defaultValue')
+    let transactionId = this.props.navigation.getParam('transactionId', 'defaultValue')
+    this.setState({ 
+      hasCameraPermission: status === 'granted',
+      customerId: userId,
+      transactionId: transactionId
+    });
   }
 
   // Runs the expo-camera takePictureAsync function to generate the image binary to upload.
   // Also pauses the preview so that the user can see what they are uploading
   // @TODO: Johnathan add functionality for user to confirm the image upload.
   snap = async () => {
     if (this.camera) {
-      this.camera.takePictureAsync({ base64: true, quality: 0 }).then(photo => {
-        this.camera.pausePreview();
-        const base64 = `data:image/jpeg;base64,${photo.base64}`;
+      this.camera.takePictureAsync({base64: true, quality: 0}).then(photo => {
+        this.camera.pausePreview()
+        let base64 = `data:image/jpeg;base64,${photo["base64"]}`
         this.setState({
           uploadStatus: 'Uploading...'
-        });
-        this._uploadImage(base64);
-      });
+        })
+        this._uploadImage(base64)
+      })
     }
   };
 
   // Uploads the base64 binary image to cloudinary and then uses that generated URL
   // to create an Airtable record with the image.
-  _uploadImage = async imageBinary => {
-    const apiUrl = 'https://api.cloudinary.com/v1_1/dxecwnkxx/image/upload';
-    const data = {
-      file: imageBinary,
-      upload_preset: IMG_KEY
-    };
+  // TODO: @Johnathan, move the airtable portion to the other file
+  _uploadImage = async (imageBinary) => {
+    let apiUrl = 'https://api.cloudinary.com/v1_1/dxecwnkxx/image/upload';
+    let data = {
+      "file": imageBinary,
+      "upload_preset": IMG_KEY,
+    }
     fetch(apiUrl, {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json'
       },
       body: JSON.stringify(data)
     })
-      .then(resp => {
-        return resp.json();
-      })
-      .then(resp => {
-        const postUrl = resp.secure_url;
-        return BASE('images').create(
-          [
-            {
-              fields: {
-                Attachments: [
-                  {
-                    url: postUrl
-                  }
-                ]
+    .then(data => {
+      return data.json()
+    }).catch(err => {
+      console.error(err)
+    })
+    .then(data => {
+      let postUrl = data.secure_url
+      let currentDate = new Date()
+      return BASE('Receipts').create([
```

Comment:
Nit: does this mean we can delete the `images` table now?

Reply:
yeah! images was just for testing -- receipts are where they are supposed to live

Reply:
Ok! It is gone now (^: